test/grpc-creds/generate.sh

```diff
@@ -21,5 +21,7 @@ for SERVICE in publisher nonce ra ca sa va rva ; do
   minica -domains "${SERVICE}.boulder,${SERVICE}1.boulder,${SERVICE}2.boulder"
 done
 
+minica -ip-addresses 10.77.77.77,10.88.88.88
```

Comment:
It's slightly confusing that this cert's filename ends up named after just one of these two IP addresses; makes it easy to forget that it actually contains both identifiers. Would you mind renaming the file so it's more obvious why it works?

Reply:
This is also the case for e.g. `ra.boulder`, which is also valid for `ra1.boulder` and `ra2.boulder`. It's part of how minica works. I'd prefer not to add the complication of renaming the file.